change/@ni-nimble-blazor-d4f88529-2aa9-48d9-b605-1eb45277fd1e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Re-enable Blazor linting",
```

Comment:
Maybe this should be a patch change since the compiler used to build the assembly will change? The comment should reflect that if so.

Reply:
Not sure if there will be any assembly changes. I wouldn't expect really expect any since the `targetFramework` of the NimbleBlazor csproj was already `net6.0`. But using `patch` since build tooling changed seemed reasonable so changed that (and mentioned the specific change in the comment, i.e. specifying .NET 6 with the global.json).